canaries/example/build.gradle

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    ext.kotlin_version = "1.6.0"
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:4.1.3"
```

Comment:
Yeah, it is set to use older gradle version because it was mentioned in the docs. We try to mirror as much as possible as what is mentioned in the getting started workflow